css/index.css

```
@@ -0,0 +1,309 @@
+body {
+  background-color: #f2f6f9;
+  margin: 0;
+  font-family: "Poppins";
+  font-weight: 500;
+}
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  background-color: #f2f6f9;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.btn-back {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 8px;
+  border: 1px solid #081c3933;
```

noone understanding what it is )
i recommend to rgb(a) for need instead of hex and this type of format
```suggestion
  border: 1px solid rgb(8, 28, 57, .2);
```